contribs/gmf/examples/style/mobileapp-search.less

```diff
@@ -0,0 +1,44 @@
+.tt-menu {
+  position: fixed!important;
+  left: 0;
+  right: 0;
+  top: 50px!important;
+  bottom: 0;
+  overflow-x: hidden;
+  overflow-y: auto;
+  width: 100vw;
+  background-color: white;
+  border: solid 1px black;
+  border-top: none;
```

Comment:
I'd prefer a border all around the tt-menu element and border-bottom for each item in the list.